sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosChangeFeedTable.scala

```diff
@@ -66,11 +66,12 @@ private class CosmosChangeFeedTable(val transforms: Array[Transform],
   private val clientConfig = CosmosAccountConfig.parseCosmosAccountConfig(effectiveUserConfig)
   private val cosmosContainerConfig = CosmosContainerConfig.parseCosmosContainerConfig(effectiveUserConfig)
   private val changeFeedConfig = CosmosChangeFeedConfig.parseCosmosChangeFeedConfig(effectiveUserConfig)
+  private val readConfig = CosmosReadConfig.parseCosmosReadConfig(effectiveUserConfig)
   private val tableName = s"com.azure.cosmos.spark.changeFeed.items.${clientConfig.accountName}." +
     s"${cosmosContainerConfig.database}.${cosmosContainerConfig.container}"
-  private val client = new CosmosClientBuilder().endpoint(clientConfig.endpoint)
-    .key(clientConfig.key)
-    .buildAsyncClient()
+  private val client = CosmosClientCache(
+    CosmosClientConfiguration(effectiveUserConfig,
+    useEventualConsistency = readConfig.forceEventualConsistency), None)
```

Comment:
why can't we pass the `cosmosClientStateHandle`  state here?

Reply:
The handle is not created, it is created after doing the point read